content/posts/March-2021.md

```diff
@@ -40,3 +40,20 @@ weight: 77
   ([packit-service#1020](https://github.com/packit/packit-service/pull/1020))
 - Jirka fixed a bug that caused improper numbering of patches for source-git.
   ([packit#1164](https://github.com/packit/packit/pull/1164))
+
+## Week 12 (March 22th - March 26th)
+
+- Franta improved database schema. It will help us to match builds and tests together more easily and solve some UX problems.
```

Comment:
```suggestion
- Franta improved database schema. It will help us match builds and tests together more easily and solve some UX problems.
```
this seems like a better wording to me